app/Api/V1/Requests/Models/Rule/StoreRequest.php

```diff
@@ -123,7 +123,7 @@ public function rules(): array
             'triggers.*.stop_processing' => [new IsBoolean()],
             'triggers.*.active'          => [new IsBoolean()],
             'actions.*.type'             => 'required|in:'.implode(',', $validActions),
-            'actions.*.value'            => 'required_if:actions.*.type,'.$contextActions.'|ruleActionValue',
```

Comment:
Instead of adding a method to FireflyValidator, create a new class (modeled after `IsBoolean` for example) in `app/Rules`, and the convert the rule to something like:

```php
'actions.*.value' => ['required_if:actions.*.type,'.$contextActions, new IsRuleActionExpression(), 'ruleActionValue']
```